### PR TITLE
docs: upgrade 0.6.4

### PR DESCRIPTION
**Pull request by dl239**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

**Review by lumianph — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [0.6.4] - 2022-10-21
+
+### Features
+- Support a new series of built-in functions `top_n_value_*_cate_where` (#2622 @aceforeverd)
+- Support online batch computation and aggregation over a full table (#2620 @zhanghaohit)
+- Support `load_mode` and thread options for `LOAD DATA` (#2684 @zhanghaohit)
```
Comment (zhanghaohit) [inline CHANGELOG.md]: `thread`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+- Improve the documents (#2476, #2486 #2514 #2611 #2693 #2462 @michelle-qinqin, #2695 @lumianph)
+- Support running MacOC compiling jobs in the CICD workflow (#2665 @dl239)
```
Comment (zhanghaohit) [inline CHANGELOG.md]: MacOC -> MacOS

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+
+### Bug Fixes
+- Recreating index fails if it has been dropped. (#2440 @dl239)
+- The `traverse` method may get duplicate data if there are same `ts` records on one `pk` (#2637 @dl239)
+- Multiple window union will fail when compiling in batch mode (#2478 @tobegit3hub, #2561 @aceforeverd)
+- `select * ...` statement may cause inconsistent output schemas in many cases (#2660 @aceforeverd)
+- Result is incorrect if the window is specified as `UNBOUNDED PRECEDING AND CURRENT ROW EXCLUDE CURRENT_ROW` (#2674 @aceforeverd)
+- Incorrect slice offsets may lead to offline jobs hang. (#2687 @aceforeverd)
+- Other minor bug fixes (#2669 dl239, #2683 @zhanghaohit)
```
Comment (zhanghaohit) [inline CHANGELOG.md]: dl239 -> @dl239

**Review by zhanghaohit — APPROVED** [review]

**Comment by aceforeverd** [comment]
bump the version number in code via
```
./steps/prepare_release.sh 0.6.5.a0
```

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Comment by dl239** [comment]
> bump the version number in code via
> 
> ```
> ./steps/prepare_release.sh 0.6.5.a0
> ```

will update in another pr

**Pull request merged**